src/main/java/org/example/Cinema.java

```diff
@@ -0,0 +1,74 @@
+package org.example;
+
+public class Cinema {
+    private int row = 5;
```

Comment:
wartości rzędu i miejsca powinny być ustawiane przez konstruktor. Dodatkowo masz napisane w wymaganiach zadania, że: "Oczywiście niezmiennymi atrybutami miejsca są: numer i rząd." to oznacza że po ustawieniu zmiennej row i place nie można jej już modyfikować - musisz pod to też użyć odpowiedniego rozwiązania.